app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +27,14 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = Hubstats::PullRequest.belonging_to_deploy(@deploy.id)
+      @pull_request_count = Hubstats::PullRequest.belonging_to_deploy(@deploy.id)
+      @stats = {
+        pull_request_count: @pull_request_count
```

Comment:
If you don't need pull_request_count in your view, don't make it an instance variable.